//
// This file is part of the WoW++ project.
// 
// This program is free software; you can redistribute it and/or modify
// it under the terms of the GNU General Public License as published by
// the Free Software Foundation; either version 2 of the License, or
// (at your option) any later version.
//
// This program is distributed in the hope that it will be useful,
// but WITHOUT ANY WARRANTY; without even the implied warranty of
// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
// GNU General Public License for more details.
// 
// You should have received a copy of the GNU General Public License
// along with this program; if not, write to the Free Software 
// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
//
// World of Warcraft, and all World of Warcraft or Warcraft art, images,
// and lore are copyrighted by Blizzard Entertainment, Inc.
// 

syntax = "proto2";
package wowpp.proto;

message VendorItemEntry {
	required uint32 item = 1;
	optional uint32 maxcount = 2;
	optional uint64 interval = 3;
	optional uint32 extendedcost = 4;
	optional bool isactive = 5 [default = true];
}

message VendorEntry {
	required uint32 id = 1;
	repeated VendorItemEntry items = 2;
}

message Vendors {
	repeated VendorEntry vendors = 1;
}
